<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 로드</string>
   </property>
  </widget>
  <widget class="QPushButton" name="procBtn">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>701</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imgSrc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>570</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imgDst">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>570</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>600</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>원본 이미지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>600</y>
     <width>66</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>변경 이미지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>850</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>임계값 :</string>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>830</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>127</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_H">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_S">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>640</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_V">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>640</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_H2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_S2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>700</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_V2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>700</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_H">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H Min값 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_H2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H Max값 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_S">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>670</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S Min값 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_S2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>740</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S Max값 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_V">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>670</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V Min값 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_V2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>730</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V Max값 :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboThr">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>800</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>800</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>이진화방법</string>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_H3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>760</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_H4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>850</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H Max값_2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_H3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>790</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H Min값_2:</string>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_H4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>820</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>180</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="CircleBtn">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>검출</string>
   </property>
  </widget>
  <widget class="QSlider" name="hSlider_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>780</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_threshold_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>800</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>임계값 :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
